ci/kind/kind-setup.sh

```diff
@@ -264,10 +264,17 @@ networking:
 nodes:
 - role: control-plane
 EOF
+
   for (( i=0; i<$NUM_WORKERS; i++ )); do
     echo -e "- role: worker" >> $config_file
   done
 
+  # Extra port mappings are required only when we provision a single node i.e. control-plane node,
```

Comment:
It's not really clear why it's "required" though. There is little context in this comment.
something like this would make more sense:
> When only the control plane Node is provisioned (no worker Node), we configure port mappings so that the Antrea Agent and Controller running on the control plane Node can be easily accessed, including on macOS. This is useful for accessing Antrea APIs. With worker Nodes, we don't configure these port mappings: in particular, we wouldn't know on which Node the Controller is running.